gammapy/data/event_list.py

```diff
@@ -654,6 +473,232 @@ def plot_time_map(self, ax=None):
         return ax
 
 
+class EventList(EventListBase):
+    """Event list for IACT dataset
+
+    Examples
+    --------
+    To load an example H.E.S.S. event list:
+
+    >>> from gammapy.data import EventList
+    >>> filename = '$GAMMAPY_EXTRA/test_datasets/unbundled/hess/run_0023037_hard_eventlist.fits.gz'
+    >>> events = EventList.read(filename)
+    """
+
+    def __init__(self, table):
+        EventListBase.__init__(self, table)
+
+    # TODO: the following properties are also present on the `DataStoreObservation` class.
+    # This duplication should be removed.
+    # Maybe the EventList or EventListDataset should have an `observation` object member?
+    @property
+    def observatory_earth_location(self):
+        """Observatory location (`~astropy.coordinates.EarthLocation`)."""
+        return _earth_location_from_dict(self.table.meta)
+
+    @property
+    def observation_time_duration(self):
+        """Observation time duration in seconds (`~astropy.units.Quantity`).
```

Comment:
Please always add an empty line between the first summary line and the rest of the docstring.
Then the property will show up nicely in the generated HTML docs.